src/sgame/sg_admin.cpp

```diff
@@ -3680,6 +3686,56 @@ bool G_admin_changemap( gentity_t *ent )
 	return true;
 }
 
+bool G_admin_changedevmap( gentity_t *ent )
+{
+	char map[ MAX_QPATH ];
+	char layout[ MAX_QPATH ] = { "" };
+
+	if ( trap_Argc() < 2 )
+	{
+		ADMP( QQ( N_("^3changedevmap:^* usage: 3changedevmap [map] (layout)") ) );
```

Comment:
`3changedevmap` should just be `changedevmap`

Reply:
oops, fixed